src/client/dfuse/ops/open.c

```diff
@@ -152,5 +153,15 @@ dfuse_cb_release(fuse_req_t req, fuse_ino_t ino, struct fuse_file_info *fi)
 		DFUSE_REPLY_ZERO(oh, req);
 	else
 		DFUSE_REPLY_ERR_RAW(oh, req, rc);
+
+	if (oh->doh_evict_on_close) {
+		rc = fuse_lowlevel_notify_inval_entry(fs_handle->dpi_info->di_session,
+						      oh->doh_ie->ie_parent, oh->doh_ie->ie_name,
+						      strnlen(oh->doh_ie->ie_name, NAME_MAX));
+
+		if (rc != 0)
+			DFUSE_TRA_ERROR(oh->doh_ie, "inval_entry() returned: %d (%s)", rc,
+					strerror(-rc));
```

Comment:
I think this is actually unsafe, once DFUSE_REPLY_ENTRY is called there's no longer a reference on the inode so I will take copies of this before landing.